src/reducers/versions.tsx

```diff
@@ -1069,12 +1072,13 @@ export const fetchDiff = ({
       dispatch(actions.abortFetchVersion({ versionId: headVersionId }));
       dispatch(errorsActions.addError({ error: response.error }));
     } else {
-      if (!getVersionInfo(versionsState, response.id)) {
+      if (forceReloadVersion || !getVersionInfo(versionsState, response.id)) {
```

Comment:
Ah, yes, we've thought about doing this before. I think it makes sense to do so in this case.